src/plugin.ts

```diff
@@ -382,7 +382,7 @@ export class MaestroPlugin implements CompilerPlugin {
         if (!this.shouldParseFile(event.file)) {
             return;
         }
-        if (isBrsFile(event.file) && this.shouldParseFile(event.file)) {
+        if (isBrsFile(event.file)) {

```

Comment:
`this.shouldParseFile()` is literally called 3 lines above here...so no need to call it a second time.